scaladoc-js/resources/scaladoc-searchbar.css

```diff
@@ -124,34 +124,201 @@
     margin-left: auto;
 }
 
-.snippet-comment-button {
-    position: absolute;
+/* Snippets */
+
+.snippet .snippet-meta {
+    border-top: 2px solid var(--inactive-bg);
+    color: var(--inactive-fg);
+    margin-top: 10px;
+    padding-top: 10px;
+    font-size: 0.75em;
+}
+
+.snippet-meta .snippet-label {
+    font-weight: bold;
+}
+
+.snippet .buttons {
+    --icon-size: 24px;
```

Comment:
Wouldn't it be better to make it `16px` as a default? Those big icons don't really fit the rest of the site.

Reply:
If so, the size of the slider should be a bit smaller as well.